### DISCUSSION TITLE
Philosophy on closing discussions

### DISCUSSION THREAD

**Discussion by nbolton** [deskflow/deskflow#7844] [General]
Hi everyone,

I wanted to share a quick thought about our community discussions and feedback process.

One of the things that makes this community great is our openness to different perspectives. Sometimes, a topic may seem closed or settled to one person, but leaving room for others to add their thoughts can lead to unexpected insights or solutions. I believe this kind of openness makes us stronger and helps us avoid the pitfalls that other projects sometimes face when shutting down discussions prematurely.

Let’s continue fostering a space where everyone feels heard and valued—even when we disagree or think there’s "nothing left to discuss." Keeping conversations open until there’s a true consensus ensures we’re building something inclusive and thoughtful.

On that note, I'd like to keep this discussion open a while so people can feel free to add their piece. Thanks for helping to make this community collaborative and constructive! Your input matters.

**Comment by Conan-Kudo** [comment]
I think it makes sense to close a discussion if there's actionable resolution to the topic. On the other hand, GitHub Discussions is strange since it behaves a bit like a forum and an issue tracker.

Maybe it makes sense to close discussions when either actionable items in the forum of issues are generated (to link to pull requests in the future)? Open discussions and communication don't necessarily have an end state and I guess they could remain open?

**Reply by cdwmhcc** [reply]
> Open discussions and communication don't necessarily have an end state and I guess they could remain open?

I fully agree with this view. If there is an accepted answer, it can be marked as closed. If there isn't, why close it? Unless it's a meaningless discussion.

**Reply by sithlord48** [reply]
does marking closed make it harder to find ? i see by default it wants to only show `is:open`
